test/src/test/java/hudson/model/UpdateCenterMigrationTest.java

```diff
@@ -37,12 +35,12 @@ void updateCenterMigration() throws Throwable {
     private static final class CustomUpdateCenterExtension extends JenkinsSessionExtension {
 
         private int port;
-        private Description description;
+        private org.junit.runner.Description description;
 
         @Override
         public void beforeEach(ExtensionContext context) {
             super.beforeEach(context);
-            description = Description.createTestDescription(
+            description = org.junit.runner.Description.createTestDescription(
```

Comment:
Is this because the JenkinsSession class is bridging between junit 4 and 5 in the test harness? Is there no JUnit 5 API for this?